src/test/scala/com/strad/service/ServiceRoutesSpec.scala

```diff
@@ -0,0 +1,62 @@
+package com.strad.service
+
+import cats.*
+import cats.effect.*
+import cats.implicits.*
+import com.strad.ServiceRoutes
+import com.strad.repositories.UserRepo
+import com.strad.repositories.domain.User
+import com.strad.service.ModelsJson.given
+import com.strad.service.domain.UserResponse
+import io.circe.*
+import io.circe.generic.auto.*
+import io.circe.syntax.*
+import munit.CatsEffectSuite
+import org.http4s.*
+import org.http4s.circe.*
+import org.http4s.circe.CirceEntityDecoder.circeEntityDecoder
+import org.http4s.client.Client
+import org.http4s.dsl.io.*
+import org.http4s.implicits.uri
+
+class ServiceRoutesSpec extends CatsEffectSuite:
+  private val success: UserRepo[IO] = new UserRepo[IO]:
```

Comment:
@mcatalano26 you had the right idea over all here.  However, we want to inherit the UserRepo and it will need all of the methods overridden to compile.